### DISCUSSION TITLE
Anzeige ungedrosselte Modulleistung neben gedrosselter Modulleistung

### DISCUSSION THREAD

**Discussion by fdienesch** [hoylabs/OpenDTU-OnBattery#608] [Ideas]
Hallo!
Wäre es möglich eine zusätzliche Anzeige mit den ungedrosselten Modulleistungen zu programmieren?
Mich würde im gedrosselten Zustand interessieren wieviel trotzdem von den Modulen kommt.
Ich hoffe es ist verständlich.
LG Chris

**Comment by schlimmchen** [comment]
> Ich hoffe es ist verständlich.

Nein, leider nicht, jedenfalls für mich nicht.

Wenn die deine Module direkt am Hoymiles Wechselrichter angeschlossen hast, und dieser ein Leistungslimit hat, dann wirst du die Leistung, die die Modul erbringen könnten ohne dieses Limit, nur bestimmen können, indem du das Limit entfernst.

Wenn deine Module an einem Laderegler hängen, der "unbegrenzt" in eine Batterie einspeist, dann kennt der Laderegler die Leistung, die die Module erbringen. Wenn es ein Victron Laderegler ist, der angeschlossen wurde, dann steht diese Zahl auch im Live View oben.
